core/src/test/scala/com/broadinstitute/dsp/Generators.scala

```diff
@@ -44,6 +44,19 @@ object Generators {
     bucketName <- genGcsBucketName
   } yield InitBucketToRemove(project, Some(InitBucketName(bucketName.value)))
 
+  val genKubernetesCluster: Gen[KubernetesCluster] = for {
```

Comment:
I can move them - will create a wb-libs PR

Reply:
this object is actually defined in `ResourceValidator` so what I did was move all the object generators for objects that are defined in workbench-libs to workbench-libs and left this generator here since `KubernetesCluster` is a `ResourceValidator` object

Reply:
thanks!